lib/cog/chat/hipchat/template_processor.ex

```diff
@@ -24,6 +24,8 @@ defmodule Cog.Chat.HipChat.TemplateProcessor do
     do: "<code>#{text}</code>"
   defp process_directive(%{"name" => "fixed_width_block", "text" => text}),
     do: "<pre>#{text}</pre>"
+  defp process_directive(%{"name" => "link", "text" => text, "url" => url}),
```

Comment:
They did, there may be some changes with the new paragraph parsing bits, but I'll address that once those things are merged.